Mage/src/main/java/mage/abilities/effects/common/ExileSpellWithTimeCountersEffect.java

```diff
@@ -38,7 +38,7 @@ public ExileSpellWithTimeCountersEffect copy() {
     @Override
     public boolean apply(Game game, Ability source) {
         Player controller = game.getPlayer(source.getControllerId());
-        Card card = game.getStack().getSpell(source.getId()).getCard();
+        Card card = game.getStack().getSpell(source.getSourceId()).getCard();
```

Comment:
can you explain this change? do existing usages still work as intended?

Reply:
It wasn't working properly with targets that weren't on the stack.  I actually changed it to 
Card card = game.getCard(source.getSourceId());
which allowed it to work with cards that aren't on the stack (Mainly epochrasite).  I tested it with all previous usuages I could find and  they still work as intended (shown below).

![image](https://github.com/magefree/mage/assets/48893494/b031b630-8362-4e03-91a7-25fb9d924acf)